tiptapy/__init__.py

```diff
@@ -69,6 +69,14 @@ def inner_render(self, node) -> str:
         return html
 
 
+class Embed(BaseContainer):
+    type = "embed"
+
+    def inner_render(self, node):
+        attrs = node['attrs']
+        return attrs.get('html', [])
```

Comment:
If `html` key is present a string is returned and empty list in it's absence.
@Pradhvan do you want to to revisit this?